src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java

```diff
@@ -20,22 +20,22 @@ public class InputRequireThisTryWithResourcesOnlyOverlappingFalse implements Aut
 
     void oneResource() {
         try (BufferedReader fldBufferedReader
-                     = new BufferedReader(new InputStreamReader(null, "utf-8"))) { } // ok
+                     = new BufferedReader(new InputStreamReader(null, "utf-8"))) { }
         catch (IOException e) { }
```

Comment:
I don't see removal of suppression in this PR.
Please recheck what you pushed

Reply:
@romani done, please review PR

Reply:
@abhishekmaity , please do not resolve review items, just reply "done" to each and every review items.